sonarqube/tests/test_config.py

```diff
@@ -0,0 +1,198 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.base import ConfigurationError
+
+pytestmark = [pytest.mark.unit]
+
+
+class TestParseConfig:
```

Comment:
Can you comment here if there's anything new or is just moved over test_unit?

Reply:
Nothing new in config tests, I have only differentiated unit tests from configuration tests